### PR TITLE
XBT NRT - QC falgs are GTSPPE (not IODE) -> not IMOS compliant anymore

### PR DESCRIPTION
**Pull request by lbesnard**
NetCDF files were created with IODE flags. Bec Cowley noticed that the flags were actually GTSPPE flags. These flags aren't part of the IMOS convention, so the NetCDF file aren't compliant anymore. This needs to be reflected in the chef-private databags

### CODE REVIEW

File: aodndata/soop/soop_xbt_nrt.py [hunk 1]
```diff
@@ -366,6 +378,24 @@ def netcdf_writer(profile_data, output_dir):
     template.variables['TEMP_quality_control']['_data'] = np.int8(profile_data['profile_data']['glob_gtspp_temp'])
     template.variables['DEPTH_quality_control']['_data'] = np.int8(profile_data['profile_data']['glob_gtspp_depth'])
 
+    ####################################################################################################################
+    ####################################################################################################################
+    ## In the following part, we modify the qc flags which are GTSPP QC flags and not IODE ones automatically added as
+    ## part of the IMOS netcdf-writer template. We also remove the IMOS part in the conventions gatt since the files
+    ## aren't IMOS compliant anymore. This part may disappear in the future, and the unittests will have to be modified
+    template.global_attributes.update({'Conventions': "CF-1.6"})
```
Comment (mhidas) [inline aodndata/soop/soop_xbt_nrt.py:386]: The whole point of templates is that you don't have to do this in code!
All you need to do is update these attribute in your [template file](https://github.com/aodn/python-aodndata/blob/fe3bc678ab91d7af4dbf757109019526bee37024/aodndata/templates/soop_xbt_nrt_bufr_nc_template.json).

Comment (lbesnard) [reply aodndata/soop/soop_xbt_nrt.py:386]: thanks. completely forgot about that

File: aodndata/soop/soop_xbt_nrt.py [hunk 2]
```diff
@@ -42,7 +42,6 @@
         'XBT_line': 1080,
         'ship_transect_number': 5036,
         'agency_in_charge': 1036,
-        'Height_of_XBT_launcher': 22177,
```
Comment (lbesnard) [inline aodndata/soop/soop_xbt_nrt.py:45]: duplicate attribute removed

**Comment by lbesnard** [comment]
thanks @mhidas , fixed now

**Comment by mhidas** [comment]
:+1: Is it ready to merge?

**Review by mhidas — APPROVED** [review]

**Pull request merged**